assessment_software/src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="navbar-container">
+    <div class="options-container">
+        <a *ngFor="let t of title; index as i"
+            [routerLink]="[url[i]]" 
+            [ngClass]="i===0?'selectedLink':''"
+            routerLinkActive="selectedLink"
+            >{{t}}
+        </a>
+    </div>
+    <div class="avatar-container">
+        <app-profile-avatar></app-profile-avatar>
```

Comment:
add some content in the avatar to see how it behaves with the navbar. currently, there drop down doesn't work since we don't have anything to show.